README.md

```diff
@@ -39,7 +39,8 @@ $ pip install tap-asana
   "client_secret": "xxx",
```

Comment:
@harshpatel4crest Code coverage change is not added in this PR

Reply:
It is added in `config.yml` file.